images/branding/logo-cms-coloured.svg

```diff
@@ -1,37 +1,35 @@
-	C143,46,134.9,54.1,124.9,54.1"/>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 72.1 72.1" style="enable-background:new 0 0 72.1 72.1;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#80BB42;}
+	.st1{fill:#3399CC;}
+	.st2{fill:#55B9A9;}
+	.st3{fill:#DD5172;}
+</style>
+<path id="XMLID_135_" class="st0" d="M-54.3,35.8l-6.2,2.2c-0.2,0.1-0.3,0.3-0.2,0.5l4.7,8.2c0.5,0.8,0.9,0.7,1-0.2l1.2-10.3
+	C-53.8,35.9-54,35.7-54.3,35.8"/>
+<path id="XMLID_132_" class="st0" d="M-61.8,0c-19.9,0-36.1,16.1-36.1,36.1c0,19.9,16.1,36.1,36.1,36.1c19.9,0,36.1-16.1,36.1-36.1
+	C-25.7,16.1-41.9,0-61.8,0 M-52.9,56.4c-0.1,1.2-0.7,1.3-1.3,0.3l-16.1-28c0-0.1-0.2-0.2-0.2-0.2l-14.8-3c-0.2,0-0.2-0.3,0-0.4
+	l36-4.1c0.2,0,0.4,0.2,0.4,0.4L-52.9,56.4z"/>
+<path id="XMLID_164_" class="st1" d="M36.9,23.8c-0.3-0.4-0.8-0.4-1.1,0L21,42.3c-0.5,0.6-0.3,1.2,0.6,1.2h29.6
+	c0.8,0,1.1-0.5,0.6-1.2L36.9,23.8z M36.4,40.3c0,0.3-0.2,0.5-0.5,0.5h-9.3c-0.6,0-0.8-0.4-0.4-0.9l9.3-11.6c0.5-0.7,1-0.5,1,0.3
+	L36.4,40.3L36.4,40.3z"/>
+<path id="XMLID_161_" class="st1" d="M36.4,0C16.5,0,0.3,16.1,0.3,36.1c0,19.9,16.1,36.1,36.1,36.1c19.9,0,36.1-16.1,36.1-36.1
+	C72.4,16.1,56.3,0,36.4,0 M58.3,47.5H14.4c-0.8,0-1.1-0.5-0.6-1.2l22-27.6c0.3-0.4,0.8-0.4,1.1,0l22,27.6
+	C59.4,46.9,59.2,47.5,58.3,47.5"/>
+<path id="XMLID_275_" class="st2" d="M136.1,22c-7.8,0-14.1,6.3-14.1,14.1s6.3,14.1,14.1,14.1c3.1,0,6-1,8.3-2.7l-5.4-4.9
+	c-0.1-0.1-0.4-0.1-0.5,0l-1.3,1.3c-0.4,0.4-0.8,0.3-0.9-0.3l-1.5-8c-0.1-0.4,0.2-0.7,0.6-0.6l8,1.5c0.6,0.1,0.7,0.5,0.3,0.9
+	l-1.3,1.3c-0.1,0.1-0.1,0.4,0,0.5l4.9,5.5c1.8-2.4,2.9-5.3,2.9-8.5C150.2,28.3,143.8,22,136.1,22"/>
+<path id="XMLID_272_" class="st2" d="M136.1,0c-20,0-36.1,16.1-36.1,36.1c0,19.9,16.1,36,36.1,36c19.9,0,36-16.1,36-36
+	C172.1,16.1,156,0,136.1,0 M153.8,51.9c0.2,0.3,0.2,0.7,0,1l-1,1c-0.3,0.3-0.7,0.3-1,0l-4.3-3.9c-3.1,2.5-7.1,4-11.4,4
+	c-10,0.1-18.1-8-18.1-17.9c0-10,8.1-18.1,18.1-18.1s18.1,8.1,18.1,18.1c0,4.4-1.6,8.4-4.2,11.5L153.8,51.9z"/>
+<path id="XMLID_293_" class="st3" d="M225.5,38.5c-0.3-0.4-0.8-0.4-1.1,0l-7.1,8.9c-0.1,0.2-0.1,0.4,0.1,0.5
+	c0.5,0.3,1.1,0.7,1.7,0.9c0.1,0.1,0.3,0,0.4-0.1l4.9-6.2c0.3-0.4,0.8-0.4,1.1,0l4.9,6.2c0.1,0.1,0.3,0.2,0.4,0.1
+	c0.6-0.3,1.2-0.6,1.7-0.9c0.2-0.1,0.2-0.4,0.1-0.5L225.5,38.5z"/>
+<path id="XMLID_292_" class="st3" d="M223.9,22c-7.1,0.5-12.8,6.4-13.1,13.5c-0.1,3.8,1.2,7.2,3.5,9.8c0.2,0.2,0.4,0.2,0.6,0
+	l9.5-11.9c0.3-0.4,0.8-0.4,1.1,0l9.5,11.9c0.1,0.2,0.4,0.2,0.6,0c2.2-2.5,3.5-5.7,3.5-9.3C239,28,232.1,21.4,223.9,22"/>
+<path id="XMLID_289_" class="st3" d="M224.9,0c-19.9,0-36.1,16.1-36.1,36.1c0,19.9,16.1,36.1,36.1,36.1S261,56,261,36.1
+	C261,16.1,244.8,0,224.9,0 M224.9,54.1c-10,0-18.1-8.1-18.1-18.1s8.1-18.1,18.1-18.1S243,26,243,36S234.9,54.1,224.9,54.1"/>
+</svg>
```

Comment:
Should we minify this whilst we're updating it? I used to use this tool https://jakearchibald.github.io/svgomg/, you can configure the amount of metadata you want in there and strip out the rest to reduce the file size.

Running this through the tool with the default settings yields a small (but relatively large) saving:

```1.19k → 945 bytes 77.46%```

Reply:
Nice one @mikedevelops wasn't aware there was such a tool around. I will use it for this ticket but there are more images like this one. Could you please create a separate generic ticket based on your comment?